app/views/map/popup.html.erb

```diff
@@ -0,0 +1,21 @@
+<p class="small">
+  <%= link_to @school.name, school_path(@school) %>
+</p>
+<p class="small">
+  <%= I18n.t('maps.school_features.school_type') %>: <%= @school.school_type.humanize %>
+</p>
+<p class="small">
+  <%= I18n.t('maps.school_features.fuel_types') %>:
+  <% if @school.has_electricity? %>
+    &nbsp;<i class='fas fa-bolt'></i>
```

Comment:
Why not use the icon component here?
`<%= render IconComponent.new(fuel_type: fuel_type) %>`